app/src/main/res/layout/recycler_layer_list_layout.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+     android:layout_height="match_parent"
+    android:layout_width="match_parent"
+    android:orientation="vertical">
+
+    <Button
```

Comment:
Probably not needed but up to you for now lol Hint in future mockups it won't be there